orders/src/main/proto/catalog.proto

```
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package com.sivalabs.bookstore.catalog.v1;
```

Buf (1.57.2)

Files with package "com.sivalabs.bookstore.catalog.v1" must be within a directory "com/sivalabs/bookstore/catalog/v1" relative to root but were in directory "orders/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)